src/index.ts

```diff
@@ -39,7 +35,7 @@ export default function vitePluginLqip(options?: LQIPPluginOptions): Plugin {
 				return null;
 			}
 
-			const img = sharp(fileURLToPath(new URL(base!, `file://${viteConfig.root}/`)));
+			const img = sharp(fileURLToPath(new URL(`file://${base!}`)));
```

Comment:
Actually I think we can simplify this even further:

```suggestion
			const img = sharp(base);
```